src/usps_processor/zip_lookup.clj

```diff
@@ -13,11 +13,12 @@
               (map (partial apply str)
                    (partition-by #{\space \-} s)))))
 
-(def zipcode->city-state
-  (let [data (-> "zipcode-city-state.edn"
+(def zipcode->city-state-timezone
+  (let [data (-> "zip_city_state_timezone.edn"
                  clojure.java.io/resource
                  slurp
                  edn/read-string)]
     (into {}
-          (for [[zip {:keys [city state]}] data]
-            [zip {:city (proper-case city) :state state}]))))
+          (for [[zip {:keys [city state timezone-id]}] data]
+            [zip {:city (proper-case city) :state state
+                  :timezone-id timezone-id}]))))
```

Comment:
Possibly, but that wasn't a substantive change with this PR, assuming you're referring to the use of `proper-case`, and we're far removed in time (and possibly personnel) from the time that decision was made. I'm all for cleaning up technical debt when we can, but this seems like a good time to put in a card to deal with it later when we have free time.